Formula/sftpgo.rb

```diff
@@ -1,9 +1,12 @@
 class Sftpgo < Formula
   desc "Fully featured SFTP server with optional HTTP/S, FTP/S and WebDAV support"
   homepage "https://github.com/drakkan/sftpgo"
-  url "https://github.com/drakkan/sftpgo/releases/download/v2.3.4/sftpgo_v2.3.4_src_with_deps.tar.xz"
```

Comment:
A Git clone is a really expensive way to get version information -- is there any way you could include a file with the right Git hash when you generate `sftpgo_vx.y.z_src_with_deps.tar.xz` that we could just read and pass as a command line argument?

(Yes, I know many other Go-based formulae do it too, and I'm not happy about those either.)

Reply:
I'll do something like this:

```
echo "${SFTPGO_VERSION}" > VERSION.txt
echo "${GITHUB_SHA::8}" >> VERSION.txt
tar cJvf sftpgo_${SFTPGO_VERSION}_src_with_deps.tar.xz *
```

any suggestions to read the second line from the VERSION.txt file trimming the space? Sorry if I'm lazy :smile: thanks

Reply:
```ruby
git_sha = (buildpath/"VERSION.txt").read.lines.second.strip
```